components/autofill/src/db/schema.rs

```diff
@@ -98,13 +98,13 @@ impl ConnectionInitializer for AutofillConnectionInitializer {
         Ok(db.execute_batch(CREATE_SHARED_SCHEMA_SQL)?)
     }
 
-    fn upgrade_from(&self, db: &Transaction<'_>, version: u32) -> Result<()> {
```

Comment:
tbh I don't think this is an improvement as the new name is vague. If you feel strongly about it, I think it should be named `upgrade_to`